app/models/manageiq/providers/kubernetes/container_manager/event_catcher_mixin.rb

```diff
@@ -47,14 +47,15 @@ def monitor_events
     reset_event_monitor_handle
   end
 
-  def process_event(event)
+  def queue_event(event)
     event_data = extract_event_data(event)
-    if event_data.nil?
-      _log.debug "#{log_prefix} Discarding event [#{event_data}]"
-    else
-      _log.info "#{log_prefix} Queuing event [#{event_data}]"
-      EmsEvent.add_queue('add_kubernetes', @cfg[:ems_id], event_data)
-    end
+    _log.info "#{log_prefix} Queuing event [#{event_data}]"
+    EmsEvent.add_queue('add_kubernetes', @cfg[:ems_id], event_data)
+  end
+
+  def filtered?(event)
+    event_data = extract_event_data(event)
```

Comment:
Ah @gauravaradhye I think you and @durandom are talking about two different things, sort of ...

@durandom seems to be saying that it would be nice to just collapse the two lines in this method to one line (but leave the method in tact).  Then, separately, maybe the `event_data` could be a separate object that's passed around (i.e., changing the method signature).

I don't *think* that @durandom is suggesting to do away with this method by collapsing *and* inlining the `filtered?` check.

So, I'd suggest that you could collapse the two lines to one in this PR.

Then, in a separate PR, look at the `event_data` refactoring possibility.